### ISSUE TITLE
meta - config files are part of git repo, so require stashing to pull

### ISSUE THREAD

**Issue by christopherreay** [grassrootseconomics/kitabu-chain#7]
Im making a branch with alterations I believe may help.

added the config files to `.gitignore` and and created `.copyAndEdit` versions.

will make pull request

can I get push auth, or shall I fork?

**Comment by christopherreay** [comment]
```
root@kitabu-dayzee-1:~/context/kitabu/kitabu-chain# git diff HEAD~1
diff --git a/.gitignore b/.gitignore
index b263945..83f3010 100644
--- a/.gitignore
+++ b/.gitignore
@@ -2,4 +2,6 @@ openethereum/.env
 openethereum/key
 openethereum/data
 openethereum/password
-devops/caddy
\ No newline at end of file
+devops/caddy
+devops/Caddyfile
+ethstats-client/.env
diff --git a/devops/Caddyfile b/devops/Caddyfile.copyAndEdit
similarity index 80%
rename from devops/Caddyfile
rename to devops/Caddyfile.copyAndEdit
index 385236d..75bfbbb 100644
--- a/devops/Caddyfile
+++ b/devops/Caddyfile.copyAndEdit
@@ -1,8 +1,8 @@
-rpc.yourdomain {
+rpc.YOURPUBLICDOMAIN {
     reverse_proxy openethereum:8545
 }
 
-ws.yourdomain {
+ws.YOURPUBLICDOMAIN {
     @websockets {
         header Connection *Upgrade*
         header Upgrade    websocket
diff --git a/ethstats-client/.env b/ethstats-client/.env.copyAndEdit
similarity index 85%
rename from ethstats-client/.env
rename to ethstats-client/.env.copyAndEdit
index c99edd5..f668fbc 100644
--- a/ethstats-client/.env
+++ b/ethstats-client/.env.copyAndEdit
@@ -1,5 +1,5 @@
 # Change this to your node identifier
-INSTANCE_NAME=
+INSTANCE_NAME=YOURARBITRARYINSTANCENAME
 # If running a custom setup i.e. as a system daemon or kubernetes pod, point it to the correct RPC endpoint
 RPC_HOST=http://openethereum:8545
 WS_SECRET=C54hHd3j

```

**Comment by kamikazechaser** [comment]
Forking is fine.

**Comment by christopherreay** [comment]
Hah thats interesting. Ill remember that. I was fiddling with the readme

**Comment by christopherreay** [comment]
closed on pull request #9
